meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/util/ZkMetaClientUtil.java

```diff
@@ -74,22 +78,24 @@ private static Map<org.apache.helix.metaclient.api.Op.Type, Function<org.apache.
 
       opmap.put(org.apache.helix.metaclient.api.Op.Type.CREATE, op -> {
         try {
-          CreateMode mode = convertMetaClientMode(((org.apache.helix.metaclient.api.Op.Create) op).getEntryMode());
-          return Op.create(op.getPath(), ((org.apache.helix.metaclient.api.Op.Create) op).getData(), DEFAULT_ACL, mode);
+          CreateMode mode = convertMetaClientMode(
+              ((org.apache.helix.metaclient.api.Op.Create) op).getEntryMode());
+          return Op.create(op.getPath(), ((org.apache.helix.metaclient.api.Op.Create) op).getData(),
+              DEFAULT_ACL, mode);
         } catch (KeeperException e) {
           throw translateZkExceptionToMetaclientException(ZkException.create(e));
         }
       });
 
-      opmap.put(org.apache.helix.metaclient.api.Op.Type.DELETE,
-          op -> Op.delete(op.getPath(), ((org.apache.helix.metaclient.api.Op.Delete) op).getVersion()));
+      opmap.put(org.apache.helix.metaclient.api.Op.Type.DELETE, op -> Op
+          .delete(op.getPath(), ((org.apache.helix.metaclient.api.Op.Delete) op).getVersion()));
```

Comment:
One suggestion for you. If this is just style change or reformat, better do it with refactor PR. Otherwise, we need to take a very deep understand what's changed but actually just format change.